scripts/test/integration-dbt-async.sh

```diff
@@ -37,13 +37,6 @@ else
     airflow db init
 fi
 
-if [ "$DBT_VERSION" = "1.7" ]; then
```

Comment:
We no longer need this as the issue does not arise on latest 1.7 version with our change above to use latest dbt adapter for that specific minor.